docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/dockerhub/index.md

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_label: Docker Hub
+description: How to configure Docker Hub webhooks with Kargo for automated artifact discovery.
+---
+
+# Docker Hub Webhook Receiver
+
+The Docker Hub Webhook Receiver responds to `push` events originating from
+Docker Hub repositories.
+
+In response to a `push` event, the receiver "refreshes" `Warehouse` resources
+subscribed to the corresponding Docker Hub repository.
```

Comment:
In contrast to some other platforms, I don't see any place in the Docker Hub docs where the webhooks are associated with any specific event names/types. So the choice to monospace `push` is incorrect. I would probably not even call it a "push event" at all. I would probably use more generic wording like "when container images or charts are pushed to a Docker Hub repository" to describe the event.